CRM/Admin/Form/Preferences/Mailing.php

```diff
@@ -22,6 +22,7 @@ class CRM_Admin_Form_Preferences_Mailing extends CRM_Admin_Form_Preferences {
 
   protected $_settings = [
     'profile_double_optin' => CRM_Core_BAO_Setting::MAILING_PREFERENCES_NAME,
+    'not_reply_email_address' => CRM_Core_BAO_Setting::MAILING_PREFERENCES_NAME,
```

Comment:
I think using `no` is more accurate here than `not`, unless you change it to `do_not_reply_email_address`